app.js

```diff
@@ -2,15 +2,48 @@
 // import { getRandomItem } from './utils.js';
 
 /* State */
+let gameState = 'guess'; // 'guess' or 'results'
+let guess = ''; // 'shell-1', 'shell-2', or 'shell-3'
+let reveal = ''; // 'pearl-1', 'pearl-2', or 'pearl-3'
+let results = ''; // 'Win!' or 'Lose'
+
+let lift = 0;
+let wins = 0;
 
 /* Actions */
-function loadPage() {}
+function loadPage() {
+    displayShells();
+    displayResults();
+    displayScoreboard();
+}
 
 /* Components */
+function displayShells() {
+    guess1.classList.remove('win', 'lose', 'hidden');
+    guess2.classList.remove('win', 'lose', 'hidden');
+    guess3.classList.remove('win', 'lose', 'hidden');
+}
```

Comment:
this is a good start, but you want to use the state (starts as 'guess') to decide which classes to add and remove to display the correct items on the page:
```
if (gameState === 'guess') {
    // add and remove classes to show what we want to show on the state view
}
```
Also you'll need to be adding and removing classes to the shells and pearls to get them to render the way you want